packages/os/os.spec

```diff
@@ -72,7 +73,9 @@ Requires: %{_cross_os}thar-be-updates
 Requires: %{_cross_os}updog
 
 %if %{_is_k8s_variant}
+%if %{_is_aws_variant}
 Requires: %{_cross_os}pluto
+%endif
 Requires: %{_cross_os}static-pods
 %endif
```

Comment:
Would it be weird to use indentation for nested ifs in this language?

Reply:
Yep, not common.

Reply:
:(